js/src/Map.js

```diff
@@ -91,6 +91,7 @@ export class PlainmapView extends ConfigurationMixin(widgets.DOMWidgetView) {
                 this._viewEvents(google);
 
                 this.layerViews.update(this.model.get('layers'));
+                this.map.setTilt(this.model.attributes.tilt)
```

Comment:
This should probably be moved to the `readOptions` method where we define the initial map options.

Normally, we read model attributes with: `this.model.get('tilt')`, not `this.model.attributes.tilt`.